core/survey/_survey/surveyNodeDefs.js

```diff
@@ -117,7 +117,7 @@ export const getNodeDefAreaBasedEstimate = (nodeDef) => (survey) =>
   getNodeDefsArray(survey).find((_nodeDef) => NodeDef.getAreaBasedEstimatedOf(_nodeDef) === NodeDef.getUuid(nodeDef))
 
 export const getAreaBasedEstimatedOfNodeDef = (nodeDef) => (survey) =>
-  getNodeDefsArray(survey).find((_nodeDef) => NodeDef.getAreaBasedEstimatedOf(nodeDef) === NodeDef.getUuid(_nodeDef))
+  getNodeDefByUuid(NodeDef.getAreaBasedEstimatedOf(nodeDef))(survey)
```

Comment:
iterating over all the node defs can be slow when using a big survey, if you have the uuid it's better to use getNodeDefByUuid